boppygames/Community-Game
# Main Menu

## All Submissions:

* [x] I have read the LICENSE and agree to the terms for contributing Work.
* [x] All 3rd party assets have been added to the `AssetManifest.list` and `.gitignore`
* [x] All code submitted follows the [Coding Conventions](https://github.com/boppygames/community-game/wiki/coding-conventions)

## Review Instructions

1. Make sure you have the Free UI Pack asset imported
2. Apply the patch for the Free UI Pack (`patch-apply.sh "Free UI Pack"`)
3. Make sure the main menu is working properly

## New Feature

1. [x] All new assets (including texture, scripts, etc) are within a `Features/<feature>` directory
2. [x] Any new art assets match the level of detail expected in the game (assets should be low-poly)
3. [x] Screenshots added if necessary

### Description of feature

Main menu - opens the overworld scene when the new game button is pressed.

### Screenshots

![Screenshot from 2021-07-09 12-50-11](https://user-images.githubusercontent.com/4099508/125117712-3a31a800-e0b4-11eb-85cb-1213bc25e1a3.png)
